catalog/app/containers/HomePage/index.js

```diff
@@ -27,11 +28,18 @@ const Styler = styled.div`
 
 export default class HomePage extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
   render() {
+    const { team = {} } = config;
+    const header = team.name && typeof team.name === 'string' ?
+      team.name.toUpperCase() : undefined;
```

Comment:
`toUpperCase()` might work in this particular case, but not what we want in general. Maybe we need two separate things: team ID (used in the DNS name, etc.) and a user-friendly team name.